### ISSUE TITLE
Tso3Pipeline creating an ecsv without association id

### ISSUE THREAD

**Issue by stscieisenhamer** [STScI-JWST/jwst#1480]
From the regression `test_nrc_tso3_2.py`,  the following file is created
```
jw87600024001_02101_00001_nis_calints_whtlt.ecsv
```
The actual name should be
```
jw87600024001_02101_00001_nis_calints_a3001_whtlt.ecsv
```

**Comment by hbushouse** [comment]
High priority for B7.2, because it's a bug related to level 4 requirements DMS-785 and DMS-786.

**Comment by stscieisenhamer** [comment]
Resolved by #1316
